docs/core/getting-started/install/mac.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Install for macOS 10.11 or higher
+description: Install for macOS 10.11 or higher
+keywords: .NET, .NET Core, C#, Getting Started, Acquisition, Cross Platform, macOS
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+
+# Install for macOS 10.11 or higher
+
+## 1. Install pre-requisites
```

Comment:
This doesn't say anything about uninstalling previous versions. Is that not necessary on Mac? (If not, it doesn't need to be stated. I was just noticing after reading all the others)